api/v1alpha1/dspipeline_types.go

```diff
@@ -157,6 +161,12 @@ type ScriptConfigMap struct {
 	Key  string `json:"key,omitempty"`
 }
 
```

Comment:
since we are now talking about doing a complete copy over of config map values, we don't really need anything more than a configmap name here, we don't need to define `KfpLauncherConfigMap struc`, we just need a string that poitns to the configmap name